### PR TITLE
Es6 changes eshealth

### PR DESCRIPTION
**Pull request by Phrozyn**
All nodes return the same stats now. We should return all node stats to the about page, and ensure we have visibility into their state too.

### CODE REVIEW

File: cron/healthToMongo.py [hunk 1]
```diff
@@ -102,12 +77,6 @@ def getEsNodesStats():
     jsonobj = r.json()
     results = []
     for nodeid in jsonobj['nodes']:
-        # Skip non masters and non data nodes since it won't have full stats
```
Comment (pwnbus) [inline cron/healthToMongo.py]: Do these type of nodes now contain full stats where we no longer have to do this logic?

Comment (Phrozyn) [reply cron/healthToMongo.py]: Yes, it's been doing this since the move to 5.x

File: cron/healthToMongo.py [hunk 2]
```diff
@@ -6,45 +6,20 @@
...
 from configlib import getConfig, OptionParser
-from logging.handlers import SysLogHandler
 from pymongo import MongoClient
 
+from mozdef_util.utilities.logger import logger, initLogger
 from mozdef_util.utilities.toUTC import toUTC
 from mozdef_util.elasticsearch_client import ElasticsearchClient
 from mozdef_util.query_models import SearchQuery, TermMatch
 
 logger = logging.getLogger(sys.argv[0])
```
Comment (pwnbus) [inline cron/healthToMongo.py]: We need to delete this line now that we're using mozdef_util/utilities/logger

Comment (Phrozyn) [reply cron/healthToMongo.py]: will do!

Comment (Phrozyn) [reply cron/healthToMongo.py]: done!

**Pull request merged**